exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -303,4 +303,9 @@
   StringValidator IMPERSONATION_POLICY_VALIDATOR =
       new InboundImpersonationManager.InboundImpersonationPolicyValidator(IMPERSONATION_POLICIES_KEY, "[]");
 
+  /**
+   * Web settings
+   */
+  String WEB_LOGS_MAX_LINES = "drill.web.logs.max_lines";
```

Comment:
Should use system option. Anything that isn't bootstrap shouldn't be kept out of *.conf